tests/src/common/WskCli.java

```diff
@@ -107,8 +107,8 @@ public String getSubject() {
     public boolean checkExists() {
         File dir;
 
-        if (WhiskProperties.useCliDownload()) {
-            String binary = getDownloadedCliPath();
+        if (WhiskProperties.useCLIDownload()) {
```

Comment:
Could all of the wsk CLI path/dir information be settled withing the WskCli constructor?  After the constructor is called, the cliPath and cliDir would be set appropriately (python vs go, downloaded cli vs built).  That way, there would not be a need for other conditional usePython or useCLIDownload checks in other methods.